terraform/stacks/bootstrap/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "concourse_password" {}
+
+variable "default_region" {}
```

Comment:
Any reason to not just default this to `us-gov-west-1`? Especially since you provide a default in the `bootstrap_concourse` module itself